app/policies/application_policy.rb

```diff
@@ -15,11 +15,23 @@ def index?
   end
 
   def update?
-    @user.admin? ||  @record.user == @user
+    is_admin? ||  @record.user == @user
```

Comment:
We could also just do `@user&.is_admin?` and dispense with the helper method.

(similarly for other policy files with the same helper method)